tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -155,7 +155,99 @@ Verify "Notebook Images Are Building" Is Not Shown When No Images Are Building
     Wait Until All Builds Are Complete  namespace=redhat-ods-applications
     RHODS Notification Drawer Should Not Contain  message=Notebooks images are building
 
+Verify Favorite Resource Cards
+    [Tags]    ODS-389
+    [Documentation]    Verifies that there is a clickable star icon on each tile and tiles get
+    ...                sorted after adding them in favourites in list and tile view and even changing
+    ...                the sort, favourite tiles are still listed first
```

Comment:
I think it's not really clear as explanation. 

Try something like: "Verifies the item in Resource page can be marked se favourite. It checks if favorite items are always listed as first regardless the view type or sorting"

Reply:
Done